mne/source_estimate.py

```diff
@@ -1350,12 +1350,20 @@ class SourceEstimate(_BaseSurfaceSourceEstimate):
     Parameters
     ----------
     data : array of shape (n_dipoles, n_times) | tuple, shape (2,)
-        The data in source space. The data can either be a single array or
-        a tuple with two arrays: "kernel" shape (n_vertices, n_sensors) and
-        "sens_data" shape (n_sensors, n_times). In this case, the source
-        space data corresponds to "numpy.dot(kernel, sens_data)".
+        The data in source space. When it is a single array, the
+        left hemisphere is stored in data[:vertices[0]] and the right
+        hemisphere is stored in data[:-vertices[1]].
+        When data is a tuple, it contains two arrays:
+
+        - "kernel" shape (n_vertices, n_sensors) and
+        - "sens_data" shape (n_sensors, n_times).
+
+        In this case, the source space data corresponds to
+        "numpy.dot(kernel, sens_data)".
```

Comment:
```suggestion
        ``np.dot(kernel, sens_data)``.
```
Also there are other instances where this `data` param in the same file should probably be updated. Can you get those, too? Maybe use `docdict` if they are all the same.

Reply:
I don't think they are the same. They seem to differ depending on whether it is a volume source estimate or not.